ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/IntTopkKudafTest.java

```diff
@@ -155,13 +159,13 @@ public void testAggregatePerformance() {
     final int topX = 10;
     topkKudaf = new TopKAggregateFunctionFactory(topX)
         .getProperAggregateFunction(Collections.singletonList(Schema.INT32_SCHEMA));
-    final Integer[] aggregate = IntStream.range(0, topX)
-        .mapToObj(idx -> null)
-        .toArray(Integer[]::new);
+    final List<Integer> aggregate = IntStream.range(0, topX)
+        .mapToObj(Integer::valueOf)
+        .collect(Collectors.toList());
     final long start = System.currentTimeMillis();
 
     for(int i = 0; i != iterations; ++i) {
-      topkKudaf.aggregate(i, aggregate);
+      topkKudaf.aggregate(i, new ArrayList<>(aggregate));
```

Comment:
Replace with  the following to avoid copying the array within the timed loop and skewing the results.

```
final List<Integer> aggregate = new ArrayList<>();
        
    final long start = System.currentTimeMillis();

    for(int i = 0; i != iterations; ++i) {
      topkKudaf.aggregate(i, aggregate);
    }
``